src/client/Microsoft.Identity.Client/Platforms/Windows/KeyGuardManager.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+
+namespace Microsoft.Identity.Client.Platforms.Windows
+{
+    /// <summary>
+    /// Platform / OS specific logic to manage KeyGuard keys.
+    /// </summary>
+    internal class KeyGuardManager : IKeyGuardManager
```

Comment:
Consider having the interface OS agnostic and 2 implementations - one for Linux and one for Windows?